source/customization/menus.rst

```diff
@@ -1,5 +1,367 @@
+                   ]
+               },
+               { "caption": "-", "id": "clipboard" },
+               { "command": "copy", "mnemonic": "C" },
+               { "command": "cut", "mnemonic": "t" },
+               { "command": "paste", "mnemonic": "P" },
+               { "command": "paste_and_indent", "mnemonic": "I" },
+               { "command": "paste_from_history", "caption": "Paste from History" }
+           ]
+       }
+   ]
+
+
+Images
+******
+
+.. figure:: images/context_menu_default.png
+   :class: shadowed
+   :figclass: float-left
+   :figwidth: 50%
+
+   The default context menu in the editing area.
+
+.. figure:: images/context_menu_modified.png
+   :class: shadowed
+   :figclass: float-left
+   :figwidth: 50%
+
+   A modified context menu in the editing area.
+
+
+.. _menu-types:
+
+Available Menus
+===============
+
+The file name
+of a ``.sublime-menu`` file
+specifies the menu to be modified
+in the application.
+
+The following menus are available:
+
+========================   ===============
+     File/Menu Name          Description
+========================   ===============
+**Main**                   Main menu
+
+**Context**                Context menu in the editing area
+
+**Find in Files**          Appears when clicking the "…" button
+                           in the :doc:`Find in Files
+                           </search_and_replace/search_and_replace_files>` panel
+
+**Side Bar**               Context menu for each node in the sidebar
+
+**Side Bar Mount Point**   Additional context menu items
+                           for the top-level nodes in the sidebar
+
+**Tab Context**            Context menu of the tab bar
+
+**Widget Context**         Context menu of input fields
+                           in all kinds of widgets,
+                           including Command Palette, Goto Anything,
+                           the Find panels
+                           and panels opened by plugins
+========================   ===============
+
+Additionally,
+the following four menus open
+when you click the respective section
+in the status bar:
+
+- **Encoding**
+- **Line Endings**
+- **Indentation**
+- **Syntax**
+
+.. figure:: images/statusbar_menu.gif
+   :class: shadowed
+
+   Demonstration of a status bar menu.
+
+
+.. _menu-items:
+
+Menu Items
+==========
+
+A menu item can either
+invoke a command (with arguments)
+when it is selected,
+or have a submenu.
+
+The available properties are:
+
+- a command name,
+- arguments for the command,
+- an ID,
+- a caption,
+- a mnemonic and
+- a submenu.
+
+In order to function properly,
+a menu item must provide at least:
+
+- a command name,
+- a caption and a submenu,
+- just a caption, or
+- an ID (see :ref:`below <item-ids>`).
+
+When parsing a menu item,
+the following rules apply:
+
+#. A menu item with a submenu
+   cannot invoke command.
+   It also overrides the :ref:`separator caption <menu-separators>`

```

Comment:
I would change this whole second part to:

    and if the separator caption is used,
    it will be rendered as a literal hyphen.